CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.2.0
+
+- Upgrade to perch-data v0.12.0 [ Breaking: Now uses a Provider ]
+- Fix all lint errors
+- Upgrade dependencies
+
 ## 0.1.0
 
 - Initial release
```

Comment:
Good job on even having a change log 👍

Reply:
Ha! I've been burned too many times to not have one. perch-data has a lengthier one